<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAssistantClass</class>
 <widget class="QMainWindow" name="ItemAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemAssistant</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QStackedWidget" name="sidebarWidgetStack">
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="buttonPanel" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="mainWidgetStack">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_4"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOnlineHelp"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutPlugins"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Item Assistant</string>
   </property>
  </action>
  <action name="actionAboutPlugins">
   <property name="text">
    <string>About &amp;Plugins</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="icon">
    <iconset resource="itemassistant.qrc">
     <normaloff>:/ItemAssistant/Resources/Web.png</normaloff>:/ItemAssistant/Resources/Web.png</iconset>
   </property>
   <property name="text">
    <string>Check for &amp;Updates</string>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="icon">
    <iconset resource="itemassistant.qrc">
     <normaloff>:/ItemAssistant/Resources/Help.png</normaloff>:/ItemAssistant/Resources/Help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Online Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="itemassistant.qrc"/>
 </resources>
 <connections/>
</ui>
